app/src/main/java/tech/pauly/findapet/utils/BindingAdapters.java

```diff
@@ -36,32 +38,35 @@ public static void defaultSelectedItem(BottomNavigationView bottomNavigationView
 
     @BindingAdapter("imageUrl")
     public static void loadImageIntoView(ImageView view, String url) {
-        if (!TextUtils.isEmpty(url)) {
-            final Transformation transformation = new RoundedCornersTransformation(10, 0);
-            Picasso.with(view.getContext())
-                   .load(Uri.parse(url))
-                   .fit()
-                   .centerCrop()
-                   .transform(transformation)
-                   .into(view);
-        }
+        final Transformation transformation = new RoundedCornersTransformation(10, 0);
+        Picasso.with(view.getContext())
+               .load(Uri.parse(url))
+               .error(R.drawable.shape_animal_image)
```

Comment:
Dumb me thinking that Picasso wouldn't support something like this. Relying on their error state allows me to reuse views even if they don't have new images, then we fallback.